damus/Views/Relays/RelayConfigView.swift

```diff
@@ -41,6 +72,17 @@ struct RelayConfigView: View {
     
     var MainContent: some View {
         Form {
+            // jb55: profile.contacts is nil below. Not sure why.
```

Comment:
If you go to your Following from your profile, and then you go to the Relays view, this will cause a list of your following to show up at the top of the view:

<img width="372" alt="iPhone_14_Pro" src="https://github.com/damus-io/damus/assets/445882/d60f908b-1fea-412e-91a5-514baec13bde">